File: Plugins/Aspose_Tasks_Java_for_Ruby/lib/asposetasksjava/Currencies/currencycodes.rb
module Asposetasksjava
  module CurrencyCodes
    def initialize()
    	# Getting Currency Code
    	get_currency_code()

    	# Getting Currency Code
    	set_currency_code()
    end

    def get_currency_code()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate project object
        project = Rjb::import('com.aspose.tasks.Project').new(data_dir + 'test_tasks.mpp')

        puts "Currency Code: " + project.get(Rjb::import('com.aspose.tasks.Prj').CURRENCY_CODE).to_string
    end

    def set_currency_code()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate project object
        project = Rjb::import('com.aspose.tasks.Project').new(data_dir + 'test_tasks.mpp')

        project.set(Rjb::import('com.aspose.tasks.Prj').CURRENCY_CODE, "USD")

        puts "Set currency code."
    end
  end
end

File: Plugins/Aspose_Tasks_Java_for_PHP/src/aspose/tasks/WorkingWithCurrencies/CurrencyCodes.php
<?php
namespace Aspose\Tasks\WorkingWithCurrencies;

use com\aspose\tasks\Project as Project;
use com\aspose\tasks\Prj as Prj;

use java\io\FileOutputStream as FileOutputStream;
class CurrencyCodes{

    public static function run($dataDir=null){

        # Getting Currency Code
        CurrencyCodes::get_currency_code($dataDir);

    	# Getting Currency Code
    	CurrencyCodes::set_currency_code($dataDir);
    }

    public static function get_currency_code($dataDir=null){

        # Instantiate project object
        $project = new Project($dataDir . 'test_tasks.mpp');

        $prj=new Prj();
        print "Currency Code: " . (string)$project->get($prj->CURRENCY_CODE).PHP_EOL;//.to_string
    }

    public static function set_currency_code($dataDir=null){

        # Instantiate project object
        $project = new Project($dataDir . 'test_tasks.mpp');

        $prj=new Prj();
        $project->set($prj->CURRENCY_CODE, "USD");

        print "Set currency code.".PHP_EOL;

    }

}

?>